script/docker_ci.sh

```diff
@@ -12,8 +12,8 @@ ALPAKA_CI_BOOST_BRANCH="boost-${ALPAKA_BOOST_VERSION}"
 
 # runtime and compile time options
 ALPAKA_DOCKER_ENV_LIST=()
-ALPAKA_DOCKER_ENV_LIST+=("--env" "CC=${CC}")
-ALPAKA_DOCKER_ENV_LIST+=("--env" "CXX=${CXX}")
+ALPAKA_DOCKER_ENV_LIST+=("--env" "ALPAKA_CI_CC=${ALPAKA_CI_CC}")
```

Comment:
setting the C compiler could be skipped we do not use the C compiler

Reply:
The boost install script is the only one, which use the variable `ALPAKA_CI_CC`. I can check, if we can use the `ALPAKA_CI_CXX` instead.

Reply:
I use the `ALPAKA_CI_CXX` variable to select the C compiler in the `install_boost.sh` script. Therefore I removed the variable `ALPAKA_CI_CC` and also `CC`.